src/webui/components/Login/index.js

```diff
@@ -1,4 +1,4 @@
-import React, {Component} from 'react';
+import React, {Component, createRef} from 'react';
```

Comment:
💯 I just learned something new 🙂 `createRef` that's new in React 16.3.

Reply:
Yes. :)